docs/memory_limits.md

```diff
@@ -0,0 +1,33 @@
+<!--
+- Copyright (c) 2019, Arm Limited and Contributors
+-
+- SPDX-License-Identifier: Apache-2.0
+-
+- Licensed under the Apache License, Version 2.0 the "License";
+- you may not use this file except in compliance with the License.
+- You may obtain a copy of the License at
+-
+-     http://www.apache.org/licenses/LICENSE-2.0
+-
+- Unless required by applicable law or agreed to in writing, software
+- distributed under the License is distributed on an "AS IS" BASIS,
+- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+- See the License for the specific language governing permissions and
+- limitations under the License.
+-
+-->
+
+# Memory limits with Vulkan on Mali GPUs
```

Comment:
I am all in favor in have vendor specific stuff, but a few things

1. Is this the right place for this information as it really doesn't relate to any particular sample
2. It should be hierarchical like `#  Memory limits with Vulkan` ... `## Mali GPUs` that way there is a common section and then a place for (hopefully other IHV who want to add things) to put their info easily in as well

Reply:
1. I think that this is an okay place for this article to reside. `docs/` is definitely the only folder this can go in unless we tie it to a sample. Given that it is documented and linked appropriately in the main README.md, I think we can leave this here.

2. I agree with the hierarchical view and will change this accordingly. Other vendors can then feel free to push their own excerpts. Thanks 😄

Reply:
thanks, one last small nit, can we add a ToC to top of this page